### PR TITLE
Add: test for getWeeklyFluids

### LINKED ISSUE #15: Create Hydration class and methods
- For a user (identified by their userID - this is the same for all methods requiring a specific user’s data), the average fluid ounces consumed per day for all time
- For a user, how many fluid ounces they consumed for a specific day (identified by a date)
- For a user, how many fluid ounces of water consumed each day over the course of a week (7 days) - return the amount for each day

### Created Hydration Methods
`const hydrationData = new Hydration(user1, data)`

### Method
- [ ] 'userAverageFluid ()'
- [ ] 'getDayFluid(id)'
- [ ] 'getWeeklyFluids()'

### PR DESCRIPTION
**Pull request by Jedeo**
What’s this PR do?
- Added
-  `userAverageFluid ()`
 -`getDayFluid(id)`
 `getWeeklyFluids()`
Where should the reviewer start?
- check the User.js
close #15

### CODE REVIEW

File: src/User.js [hunk 1]
```diff
@@ -1,18 +1,48 @@
...
+    return firstName[0];
+  }
+
+  calAverageFluid(fuildsData) {
+    const userFuildsInfo = fuildsData.filter((user) => user.userID === this.id);
+    const average = userFuildsInfo.reduce((acc, fuild) => {
+      acc += fuild.numOunces;
+      return acc;
+    }, 0);
+    return average;
```
Comment (forsethnico) [inline src/User.js:23]: Need to update to actually calculate average and divide by array length

File: src/User.js [hunk 2]
```diff
@@ -1,18 +1,48 @@
...
+
+      return acc;
+    }, 0);
+
+    return sum;
+  }
+
+  getWeeklyFluids(data) {
+    const userInfo = data.filter((data) => data.userID === this.id);
+    const userDates = userInfo.map((dates) => {
```
Comment (forsethnico) [inline src/User.js:41]: Need to update to 7 day week time frame and filter by that

**Review by forsethnico — APPROVED** [review]
This is a great start! Need to improve a few of the methods to include average calculation and update the weekly ounces calculation to include use of 7 day/week timeframe appropriately.

**Review by piperjarrett — COMMENTED** [review]
Good job Jedeo, there are a couple spelling mistakes. Also for the average method, we need to divide by the length of the userFluidsInfo. also the last one will need to be fixed to only seven days.

**Pull request merged**